pkg/agents/team2/reward.go

```diff
@@ -0,0 +1,29 @@
+package team2
+
+func calcReward(hp int, hpInc int) float32 {
+	//TODO: reward should be redone later according to new healthInfo
+	ret := float32(0.0)
+
+	//we encourage agent to survive
+	if hp > 20 {
+		ret += 1.0
+	} else {
+		ret -= 0.5
+	}
+	//we encourage ageny to eat less when hp level is high
+	oldHP := float32(hp - hpInc)
+	incRate := float32(hpInc) / oldHP
+	if incRate > 1.0 {
+		ret += 1.0
+	} else {
+		ret -= 1.0
+	}
+	return ret
```

Comment:
```suggestion
	if incRate > 1.0 {
		return ret + 1.0
	}
	return ret - 1.0
```

Reply:
will do.

Reply:
No, actually the Reward calculation will get more sophisticated, and it is better to leave the return value at the end line. Gonna change it, but might bring it back later.